schemas/com.google.analytics.measurement-protocol/screen_view/jsonschema/1-0-0

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Google Analytics screen_view hit",
+  "self": {
+    "vendor": "com.google.analytics.measurement-protocol",
+    "name": "screen_view",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+
+  "type": "object",
+  "properties": {
+    "hitType": {
+      "enum": [
+        "screenview"
+      ]
+    },
+    "screenName": {
+      "type": "string",
+      "maxLength": 2048
+    }
+  },
+  "required": ["hitType", "screenName"],
+  "additionalProperties": false
+}
```

Comment:
Unclear as to how this relates to the app_tracking schema which seems to contain all data regarding a `screenview` hit type